xwiki-platform-core/xwiki-platform-attachment/xwiki-platform-attachment-picker/xwiki-platform-attachment-picker-webjar/src/main/webjar/attachmentGalleryPicker.js

```diff
@@ -128,14 +106,24 @@ define('xwiki-attachment-picker',
           typesFqs = [];
         }
         const computedFqs = ['type:ATTACHMENT'].concat(optionsFqs).concat(typesFqs);
-        const query = computedFqs.map((fq) => 'fq=' + fq).join('\n');
+        const computedParams = [];
+
+        computedParams.push(...computedFqs.map((fq) => 'fq=' + fq));
+
+        if (options.sort) {
+          computedParams.push('sort=' + options.sort);
+        }
+
+        const query = computedParams.join('\n');
 
         return new Promise((resolve, reject) => {
           // TODO: handle more kind of scopes
           if (input === '') {
-            // Replace the empty string with '*', matching everything. 
+            // Replace the empty string with '*', matching everything.
```

Comment:
In general, we tend to avoid minor changes like this in a PR as it make it slightly more difficult to reviews ;)
But they are definitely nice to have in a separate misc commit.

Reply:
Thank you, I'll remove the change, my editor removed the trailing space without me noticing :)